daemon/network.go

```diff
@@ -293,6 +293,16 @@ func (daemon *Daemon) createNetwork(cfg *config.Config, create types.NetworkCrea
 		return nil, PredefinedNetworkError(create.Name)
 	}
 
+	c := daemon.netController
+	driver := create.Driver
+	if driver == "" {
+		driver = c.Config().DefaultDriver
+	}
+
+	if driver == "overlay" && !daemon.cluster.IsManager() && !agent {
+		return nil, errdefs.Forbidden(errors.New("This node is not a swarm manager. Use \"docker swarm init\" or \"docker swarm join\" to connect this node to swarm and try again."))
```

Comment:
Can you change this back to use string-literal (back-tick); that way we don't need to escape the inner-quotes);

```suggestion
		return nil, errdefs.Forbidden(errors.New(`This node is not a swarm manager. Use "docker swarm init" or "docker swarm join" to connect this node to swarm and try again.`))
```

Reply:
Oh, and we probably need the `//nolint: revive`, because errors should not be capitalized

Reply:
> Oh, and we probably need the //nolint:revive, because errors should not be capitalized

Strangely, the linter doesn't pick it up :thinking:

Reply:
hm.. wonder if we disabled that part of the linter.. it's possible